tests/networkmanager/cni_test.cpp

```diff
@@ -0,0 +1,1246 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <filesystem>
+#include <fstream>
+
+#include <Poco/JSON/Parser.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <aos/test/log.hpp>
+#include <utils/json.hpp>
+
+#include "networkmanager/cni.hpp"
+
+using namespace aos;
+using namespace aos::sm::cni;
+using namespace testing;
+
+/***********************************************************************************************************************
+ * Mocks
+ **********************************************************************************************************************/
+
+class MockExec : public ExecItf {
```

Comment:
Put to include/mocks/cnimock.hpp if could be reused

Reply:
Done